Project.toml

```diff
@@ -13,6 +13,7 @@ JSON2 = "2535ab7d-5cd8-5a07-80ac-9b1792aadce3"
 LightGraphs = "093fc24a-ae57-5d10-9952-331d41423f4d"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 MetaGraphs = "626554b9-1ddb-594c-aa3c-2596fe9399a5"
+Neo4j = "d2adbeaf-5838-5367-8a2f-e46d570981db"
```

Comment:
May want to make this a conditional dependency in the future, if we want to retain RoME on RasPi constraint.  It's definitely okay for now, but probably time for drawings and pizza to figure out where we want the hard and conditional boundaries to be.